com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -106,15 +126,31 @@ internal NetworkSerializer __beginSendClientRpc(uint rpcMethodId, ClientRpcParam
         {
             PooledNetworkWriter writer;
 
+            if (clientRpcParams.Send.UpdateStage == NetworkUpdateStage.Unset)
+            {
+                clientRpcParams.Send.UpdateStage = NetworkUpdateLoop.UpdateStage;
+
+                if (clientRpcParams.Send.UpdateStage == NetworkUpdateStage.Initialization)
+                {
+                    clientRpcParams.Send.UpdateStage = NetworkUpdateStage.EarlyUpdate;
+                }
+            }
```

Comment:
ditto, encapsulate this one into a function call

Reply:
Done.